lib/supraffi/gosupra/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
Change name to 'batch' as post-beta will likely be modified code that Supranational doesn't make.

Reply:
Oh, this was just an early tool to test that the mad linker invocation works correctly, will remove